restcomm/restcomm.application/src/main/java/org/restcomm/connect/application/Bootstrapper.java

```diff
@@ -214,6 +214,7 @@ public UntypedActor create() throws Exception {
 
     private String home(final ServletContext context) {
         final String path = context.getRealPath("/");
+        logger.info("Home path:" + path);
```

Comment:
can we use guarded logging here

Reply:
sure,actually it was a logging i put to debug something, i will simply remove it.